### ISSUE TITLE
Latest version 1.50.1 fails to upload docker images to Artifactory

### ISSUE THREAD

**Issue by Kyslik** [jfrog/jfrog-cli#1147]
**Describe the bug**

TLDR: command `jfrog rt dp` fails on latest version of `jfrog-cli` - https://github.com/jfrog/jfrog-cli/releases/tag/v1.50.1

We use CircleCI in tandem with Artifactory orb (`circleci/artifactory@1.0.0`), until very recently (couple hours) builds worked however now after using `v1.50.1` (latest), built images fail to push and `jfrog` binary just exits with 1 - `Exited with code exit status 1`.

**To Reproduce**
I haven't reproduced on local machine yet will edit the GitHub issue once I do.

**Expected behavior**
Image to be uploaded/published.

**Screenshots**
n/a

**Versions**
- JFrog CLI version: 1.50.1
- JFrog CLI operating system: Ubuntu 18.04.5 LTS
- Artifactory Version: SaaS

**Additional context**

Maybe related PR: https://github.com/jfrog/jfrog-cli/pull/1106
Related blame: https://github.com/jfrog/jfrog-cli/blame/681fb5e5e50c3f306b92a718a1ce6a34fc9dbc7a/artifactory/cli.go#L1406-L1416

**Comment by vincent-chan-mox** [comment]
similar problem with logs
```2021-07-14T03:27:42.196Z [jfrt ] [INFO ] [c911fc8506db247f] [o.a.e.UploadServiceImpl:464   ] [http-nio-8081-exec-4] - Deploy to 'xxx:xxx/_uploads/20dbbcc1-3542-4e68-b5eb-bdfcef3280cb' Content-Length: 1 (estimation) artificial: false```

**Comment by Kyslik** [comment]
Related code change I could find: https://github.com/jfrog/jfrog-cli/blame/681fb5e5e50c3f306b92a718a1ce6a34fc9dbc7a/artifactory/cli.go#L1406-L1416

**Comment by Kyslik** [comment]
Could there be a revert of https://github.com/jfrog/jfrog-cli/commit/681fb5e5e50c3f306b92a718a1ce6a34fc9dbc7a?

cc @eyalbe4 @talarian1 @asafgabai (sorry to pull everyone here; just saw recent contributors thus tagged you explicitly).

**Comment by eyalbe4** [comment]
@Kyslik and @vincent-chan-mox - 
Thanks for reporting this!
We're looking into this right now.

**Comment by Kyslik** [comment]
I see new version is out; we are testing it.

**Comment by eyalbe4** [comment]
@Kyslik and @vincent-chan-mox -
Our apologies for the inconvenience caused by this bug.
We believe https://github.com/jfrog/jfrog-cli-core/pull/177 fixes the issue.
We've just released version 1.50.2 which includes this fix.
Looking forward to your to your feedback for it.

**Comment by Kyslik** [comment]
Fixed in 9502165f271b158a0dbd9b9369c53076ab01702c.
